### PR TITLE
On off flag

### PR DESCRIPTION
**Pull request by DanialErfanian**
# Description
Add a flag(settings.py) for disabling history logging by simple_history app. It's maybe useful when you want to save history from specific deployment and ignore history in others.

## Motivation and Context
In our case we need to save history for admin panel to be able track objects modified by our operation team but we don't want history of changes made by API calls.

### CODE REVIEW

File: simple_history/models.py [hunk 1]
```diff
@@ -470,12 +470,16 @@ def get_meta_options(self, model):
...
     def post_save(self, instance, created, using=None, **kwargs):
+        if not getattr(settings, "SIMPLE_HISTORY_ENABLED", True):
+            return
         if not created and hasattr(instance, "skip_history_when_saving"):
             return
         if not kwargs.get("raw", False):
             self.create_historical_record(instance, created and "+" or "~", using=using)
 
     def post_delete(self, instance, using=None, **kwargs):
+        if not getattr(settings, "SIMPLE_HISTORY_ENABLED", True):
```
Comment (rossmechanic) [inline simple_history/models.py:481]: How would this handle bulk history update and create?

**Comment by jeking3** [comment]
@DanialErfanian there's an open question that needs answering.

File: simple_history/models.py (hunk 1 above)
Comment (DanialErfanian) [reply simple_history/models.py:500]: Thanks for the comment, Fixed.

**Comment by DanialErfanian** [comment]
@jeking3 
I fixed the problem.

**Review by jeking3 — APPROVED** [review]

**Comment by DanialErfanian** [comment]
@jeking3 
Thanks for quick response, Do you have any guess for time of the next release?

**Pull request merged**